### PR TITLE
Push Job client updates

### PR DESCRIPTION
**Pull request by schisamo**
This PR along with https://github.com/opscode-cookbooks/opscode-ci/pull/265 will allow Push Jobs Client to be built on manhattan.ci.chef.co

/cc @chef/ociv @chef/client-engineers @manderson26

### CODE REVIEW

**Comment by yzl** [comment]
lgtm

File: config/projects/push-jobs-client.rb [hunk 1]
```diff
@@ -41,10 +41,22 @@
...
+# if you have this version of the CA bundle stored via S3 caching (which Chef
+# Software does).
+override :cacerts, version: '2014.08.20'
+
 override :bundler,        version: "1.7.2"
-override :ruby,           version: "1.9.3-p547"
-override :'ruby-windows', version: "1.9.3-p484"
-override :rubygems,       version: "2.4.1"
+override :ruby,           version: "2.1.4"
+override :'ruby-windows', version: "2.0.0-p451"
```
Comment (tyler-ball) [inline config/projects/push-jobs-client.rb]: The ChefDK 0.5.0 RC is using 2.1.5 for both windows and non-windows.  People haven't reported any major issues so far, if you wanna try that.

**Comment by scotthain** [comment]
:+1:

File: config/projects/push-jobs-client.rb (hunk 1 above)
Comment (schisamo) [inline config/projects/push-jobs-client.rb]: Thx @tyler-ball I'll try that out.

**Comment by schisamo** [comment]
I ran a Chef and ChefDK build through just to make sure the omnibus and omnibus-software bumps are not going to cause any issues:

http://manhattan.ci.chef.co/job/chef-build/44/
http://manhattan.ci.chef.co/job/chefdk-build/230/

**Pull request merged**